CMakeLists.txt

```diff
@@ -1297,6 +1297,27 @@ add_custom_command(
   # NB: COMMENT only works for some CMake-Generators
 )
 
+# To populate with hex files to merge, do the following:
```

Comment:
need this to be documented somewhere, not only in the cmake files.

Reply:
Do you have a proposal to where? I am not well known in where doc stuff should go.

Reply:
`doc/application/application.rst`

Reply:
I'd say here somewhere: https://docs.zephyrproject.org/latest/application/application.html

Reply:
Check it now.